draft-ietf-core-cf-reg-update.md

```diff
@@ -196,16 +196,20 @@ A new column with the title "Notes" has been added to the CoAP Content-Formats R
 
 ## New Section 12.3.1 "Temporary Content-Format Registrations"
 
-This section clarifies that the "CoAP Content-Formats" registry allows temporary registrations within the 0-255 and 256-9999 ranges.
-The range 10000-64999 does not allow temporary registrations.
+This section clarifies that the "CoAP Content-Formats" registry allows temporary registrations within the 0-255, 256-9999, and 10000-64999 ranges.
 
 A temporary registration may be created for example by an IANA early allocation action, as requested by the authors of an Internet-Draft in the IETF stream.
```

Comment:
AFAIU, it’s a bit more intricate than that: early allocations are requested by WG chairs after the AD has approved the request made by the editors.

Reply:
Right; so maybe s/as requested by the authors of an Internet-Draft/as requested for an Internet-Draft

Reply:
What about: "A temporary registration may be created for example by an IANA early allocation action, based on an Internet-Draft in the IETF stream."

This is just an example; so we don't exclude possible other ways or other types of documents.

Reply:
YAP (yet another proposal)

> _"A temporary registration may be created for example by an IANA early allocation action {{-iana-early}}.”_

Reply:
Ok, used this in the latest commit.